scheduler/scheduler.go

```diff
@@ -278,15 +278,19 @@ func (sched *Scheduler) StartNewUser(ctx context.Context, timeout time.Duration,
 }
 
 func setupRESTHandler(sessionState *session.State, connectionSettings *connection.ConnectionSettings) error {
-	headers, err := connectionSettings.GetHeaders(sessionState, "")
+	host, err := connectionSettings.GetHost()
```

Comment:
shouldn't even need the GetHost(), right?

Reply:
It's used on line 290

Reply:
wait, ok so just `_, err = connectionSettings.GetHeaders(sessionState, "")`  wouldn't be enough?

Reply:
as that one does GetHost function if host parameter is `""`

Reply:
didn't see the double usage before, had accidentally switched to unified diff in github which makes things hard  to read.

Reply:
Ah yes I see that now.